script.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+// Part 1 //
+
+let number1 = '8';
```

Comment:
Змінні які далі по ходу виконання коду не перезаписуются, треба об'являти через ключове слово const

І в змінній судячи з назви та, тому що далі проводяться математичні операції, має бути число, а чомусь по типу данних це рядок
```suggestion
const number1 = 8;
```